DownUnder.UI/UI/Widgets/Behaviors/BehaviorManager.cs

```diff
@@ -167,21 +166,27 @@ public void RemoveAt(int index)
         /// <returns> A <see cref="List{WidgetBehavior}"/> of removed <see cref="WidgetBehavior"/>s. (if any) </returns>
         public List<WidgetBehavior> RemoveIDed(string behavior_id) {
             var removed = new List<WidgetBehavior>();
-            for (int i = 0; i < _behaviors.Count; i++) {
-                if (_behaviors[i].BehaviorIDs.Contains(behavior_id)) {
-                    removed.Add(_behaviors[i]);
-                    _behaviors.RemoveAt(i--);
-                }
+
+            for (var i = 0; i < _behaviors.Count; i++) {
+                var beh = _behaviors[i];
+                if (!beh.BehaviorIDs.Contains(behavior_id))
+                    continue;
+
+                removed.Add(beh);
```

Comment:
beh

Reply:
couldn't think of a good abbreviation 8)